modules/compliance-profile-types.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/co-concepts/compliance-operator-understanding.adoc
+// * security/compliance_operator/co-scans/compliance-operator-supported-profiles.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="compliance_profile_types_{context}"]
+= Compliance Operator profile types
+
+Compliance Operator rules are organized into profiles. Profiles can target the {product-title} Platform or Nodes, and some standards include `rhcos4` Node profiles.

```

Comment:
Consider rewriting this to avoid the phrase "OpenShift Container Platform Platform".
```suggestion
Compliance Operator rules are organized into profiles. Profiles can target the Platform or Nodes for {product-title}, and some standards include `rhcos4` Node profiles.

```